packages/now-next/src/utils.ts

```diff
@@ -300,6 +300,34 @@ export function getDynamicRoutes(
     return [];
   }
 
+  try {
```

Comment:
I'm not a fan of using an error to handle control-flow for the file does not exist case. Let's unwrap this and put it behind a `exists` check.